contracts/Recruitment.sol

```diff
@@ -196,33 +197,38 @@ contract Recruitment is Ownable, ReentrancyGuard {
      * @notice Registers a referral
      * @param jobId The job ID already registered with the contract.
      * @param refereeMail The email of the referee.
+     * @param referralCode The referral code of the referee, it must be unique and hashed.
      */
     function registerReferral(
         uint256 jobId,
-        bytes32  refereeMail
+        bytes32 refereeMail,
+        bytes32 referralCode
```

Comment:
Ideally the referralCode is generated SC side?

Reply:
That's planned to get the referral code from front right now. like Jsontoken.